action/__main__.py

```diff
@@ -33,6 +33,9 @@
             database=os.environ["DB_NAME"],
             user=os.environ["DB_USER"],
             password=os.environ["DB_PASSWORD"],
+            exclude_tables=list(
+                filter(None, os.getenv("EXCLUDE_TABLES", "").split(","))
+            ),
```

Comment:
FYI this will allow somebody to pass something like "table1,,,table2,table3,,," or even ",,,,," as valid input. Is this intended behaviour or would we want to validate that each table is non-empty and throw an error before instantiating PostgresStore otherwise?

Reply:
I believe its fine, adopter of the action should be able to see that they missed something if they intended to exclude some tables ?